src/main/resources/partitions_statistics.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "t2.large",
```

Comment:
I know that this experiment result has been around for a while, but I think we are not gonna use this sorte of instance outside of our sandbox environment, probably. We are currently experimenting with the following instances types:

* M3
        - m3.large
        - m3.xlarge
        - m3.2xlarge
* C4
       - c4.large
       - c4.xlarge
       - c4.2xlarge
       - c4.4xlarge

Should I reconsider the instance types in the experiment or should we create a new ticket in order  to reevaluate the present statistics?